include/magma/vk/logicalDevice.h

```diff
@@ -31,26 +33,14 @@ class LogicalDevice
     VkQueue getGraphicsQueue() const { return graphicsQueue; }
     VkQueue getPresentQueue() const { return presentQueue; }
     const VkCommandPool &getGraphicsCmdPool() const { return graphicsCmdPool; }
-    TextureManager &getTextureManager() const { return *_textureManager; }
+    TextureManager  &getTextureManager()    const { return *_textureManager;    }
+    BufferManager   &getBufferManager()     const { return *_bufferManager;     }
 
     VkDeviceMemory createDeviceMemory(VkMemoryRequirements memRequirements, VkMemoryPropertyFlags properties);
 
-    //////////////////////////////////////////////////////////////////////////////////////////////////////
-    // buffer management
-    //////////////////////////////////////////////////////////////////////////////////////////////////////
-    Buffer createBuffer(VkDeviceSize size, VkBufferUsageFlags usage, VkMemoryPropertyFlags properties);
-    template<class T>
-    Buffer createVertexBuffer(const std::vector<T> &data);
-    template<class T>
-    Buffer createIndexBuffer(const std::vector<T> &data);
-    Buffer createUniformBuffer(VkDeviceSize size);
-
-    void deleteBuffer(Buffer &buffer);
     //////////////////////////////////////////////////////////////////////////////////////////////////////
     //
     //////////////////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
delete unnecessary comments.
There is no more logical block to be finished by these comment lines

Reply:
Done